SobokSobok/SobokSobok/Presentation/Notice/NoticeViewController.swift

```diff
@@ -56,7 +45,7 @@ extension NoticeViewController: UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = notificationTableView.dequeueReusableCell(withIdentifier: NoticeTableViewCell.cellIdentifier) as? NoticeTableViewCell else {return UITableViewCell()}
+        guard let cell = notificationTableView.dequeueReusableCell(withIdentifier: NoticeTableViewCell.reuseIdentifier) as? NoticeTableViewCell else {return UITableViewCell()}
```

Comment:
```suggestion
       let cell = notificationTableView.dequeueReusableCell(for: indexPath, cellType: NoticeTableViewCell.self)
```
저번에 설명했던것처럼 요렇게 작성가능하므로 바꿔주세요~